dtls_debug.h

```diff
@@ -24,6 +24,10 @@
 #include "global.h"
 #include "session.h"
 
+#ifdef IS_WINDOWS
```

Comment:
Not sure, why this is required?

In "dtls_debug.c" it's simply included with:

```
#include <stdio.h>
```

If the the include is really required, I would go without `#ifdef IS_WINDOWS`.